src/main/java/homework2/StringConcatenate.java

```diff
@@ -0,0 +1,8 @@
+package homework2;
+
+public class StringConcatenate {
+
+    public static String concatenateAndRemove(String firstString, String secondString) {
```

Comment:
why have you removed your algorithmic solution? :)

Reply:
now I see that my solution was not so elegant as current one, so lets keep only this :)